### DISCUSSION TITLE
Stopping generation doesn't actually stop interference

### DISCUSSION THREAD

**Discussion by Arche151** [open-webui/open-webui#2686] [General]
I am running the docker image with bundled Ollama (CPU interference) and when I click the stop button during generation, in the UI the text generation stops, but my CPU is still under as much load as before.

And when I start a new chat, the text generations takes forever. Only after restarting the Docker container I get the same speeds as before.

So clicking stop doesn't actually stop the interference. 

@tjbck I don't understand, why you turned this issue into a discussion. This is a bug, that multiple people report.

**Comment by SaturnIC** [comment]
This is still not fixed

**Comment by frostysashimi** [comment]
I seem to have mitigated this by turning off autocomplete, tag, and retrieval query generation. This is under Admin Panel > Settings > Interface.

Unfortunately I have not tested which of the 3 or all 3 features cause the issue.

Perhaps downloading and using a smaller model as the 'Local Models" might allow for keeping these functions and not murdering the CPU?
